### DISCUSSION TITLE
Looking for a mentor

### DISCUSSION THREAD

**Discussion by harpreet-singh-147** [community/community#30718] [New to GitHub]
Hello. I was wondering if anyone is available to mentor me? Or someone to contact if i am stuck (like i am right now) on anything? I will be starting a bootcamp soon, and i think i need a bit of help in preparing. The language will be Javascript. Please get in touch if this is possible. Thanks

**Comment by Zacchaeus-Oluwole** [comment]
Check this link below, It will help you and if you need a help you can message me on LinkedIn https://www.linkedin.com/in/zacchaeus-oluwole-7bba6b191/

JavaScript link https://github.com/Asabeneh/30-Days-Of-JavaScript

**Comment by jfullstackdev** [comment]
just leave your questions here too, so that others might be of help
